impeller/renderer/backend/vulkan/resource_manager_vk.cc

```diff
@@ -6,21 +6,28 @@
 
 #include "flutter/fml/thread.h"
 #include "flutter/fml/trace_event.h"
+#include "fml/logging.h"
 
 namespace impeller {
 
 std::shared_ptr<ResourceManagerVK> ResourceManagerVK::Create() {
-  return std::shared_ptr<ResourceManagerVK>(new ResourceManagerVK());
+  auto manager = std::shared_ptr<ResourceManagerVK>(new ResourceManagerVK());
```

Comment:
```suggestion
  auto manager = std::make_shared<ResourceManagerVK>(());
```

Reply:
Sorry, i had extra parenthesis in there: `auto manager = std::make_shared<ResourceManagerVK>();`

Reply:
Done.

Reply:
Ugh, you can't actually used "make_shared" because we are using `friend` to access private constructors.  I thought that might be the case after the fact, sorry.  Adding `make_shared` a friend kind of defeats the purpose so you have to switch it back.

Reply:
Yeah that's fun, reverted.